src/otx/algorithms/common/adapters/mmcv/utils/_builder_build_data_parallel.py

```diff
@@ -153,6 +154,8 @@ def scatter(self, inputs, kwargs, device_ids):
                 for val in x:
                     if isinstance(val, dict):
                         for k in val:
+                            if not self.put_gt_on_device and k.startswith("gt_"):
+                                continue
```

Comment:
It is just to use post processing on CPU, right?

Reply:
Yes, we don't need to put annotations on HPU if we compute loss on CPU. 
YOLOX from Habana team do the same -> they put post-processing on CPU. But I think it is not a good way to boost the perfomance